contest/contributions/danese-covid19-understanding.yml

```
@@ -0,0 +1,10 @@
+title: Understanding COVID-19 through Italian excess deaths analysis
+author: Alberto Danese
+abstract: |
+  Why are we expecting to understand COVID-19, when most of the analysis rely on the same biased, unreliable and dull data? Death and cases are counted differently across countries. In Italy, regions and provinces have different capabilities and policies when it comes to testing the population. All this makes drawing conclusions a hard and risky task. 
+  Here I suggest an original approach based on overall deaths data from ISTAT, available from 2015 to 2020, done 100% in R.
+  I focus on excess deaths with respect to previous years and answer many questions with a robust quantitative approach. Are men really more hit by COVID-19 than women? Is it true that only the elderly are affected? What really happened in Bergamo and other Italian provinces?
+menu_entry: Understanding COVID-19 through Italian excess deaths analysis
+repository: https://github.com/albedan/covidr-excess-deaths-analysis
+type: webpage
+content: https://bookdown.org/connect/#/apps/4119
```

We cannot embed the specified `content` as a gallery page via an `iframe`, see the minimal embedding example 
[danese-covid19-understanding.zip](https://github.com/Milano-R/erum2020-covidr-contest/files/4603302/danese-covid19-understanding.zip)
```
Load denied by X-Frame-Options: “deny” from “https://bookdown.org/connect/#/apps/4119”, site does not permit any framing. Attempted to load into “file:///home/riccardo/Downloads/danese-covid19-understanding.html”.
```
This is probably prevented by the RSConnect wrapper of the actual content, which I could retrieve as https://bookdown.org/content/4119/Covid19-Istat-Age.html
```suggestion
content: https://bookdown.org/content/4119/Covid19-Istat-Age.html
```